Tasks/UseNodeV1/installer.ts

```diff
@@ -104,6 +107,21 @@ async function queryLatestMatch(versionSpec: string): Promise<string> {
     // get the latest version that matches the version spec
     let version: string = toolLib.evaluateVersions(versions, versionSpec);
     return version;
+
+    function getRestClient() {
```

Comment:
This is where I ensure that requestOptions have all necessary configurations in place